javascript/javascript3/week3/Promises/script.js

```diff
@@ -0,0 +1,51 @@
+//////////////////////////////////////////////////////Getting into promises
+
+
+
+// Promise.all(requests)
+//     .then(responses => {
+//         // all responses are resolved successfully
+//         responses.forEach(response => response.jcon())
+//     })
+const ul = document.querySelector('ul');
+
+
+class Classmates {
+    constructor(classmate1, classmate2, classmate3) {
+        this.classmate1 = classmate1,
+            this.classmate2 = classmate2,
+            this.classmate3 = classmate3
+    }
+    promis() {
+        let names = [this.classmate1, this.classmate2, this.classmate3];
+        let requests = names.map(name => fetch(`https://api.github.com/search/repositories?q=user:${name}`));
+        Promise.all(requests).then((responses) => responses)
+            .then(responses => Promise.all(responses.map(r => r.json())))
+            .then(users => {
+                users.forEach(user => {
+                    let userName = user.items[0].owner.login;
+                    let div = document.createElement('div')
+                    div.style.fontSize = '20px';
+                    div.style.fontStyle = 'italic';
+                    div.style.margin = '20px 0 20px 0';
+                    div.style.textShadow = '7px 2px 10px blue';
+                    div.style.fontWeight = 'bold';
```

Comment:
could be great to create a style file and a class to assign it to the new div.

Reply:
thank you for your feedback. whould you please let me kno how I can make it. I've tried to make div.style.cssText but it was not helpfull. Moreover I could not define and use getElementByTagName('STYLE')[0] to attach a style to a div in the DOM.

Reply:
Hey, you can try by the following code:

```
let div = document.createElement('div');
                    div.class = 'div-repos';

```
then in the style.css file add the following:

```
.div-repos{
      font-size : 20px;
      font-style : italic;
      margin : 20px 0 20px 0;
      text-shadow : 7px 2px 10px blue;
      font-Weight : bold;
}
```